app/controllers/cars_controller.rb

```diff
@@ -16,6 +16,7 @@ def new
   def create
     @car = Car.new(cars_params)
     if @car.save
+      properties.create car: @car, user: current_user
```

Comment:
@car.user = current_user
Add this line and delete the line below
properties.create car: @car, user: current_user